deploy-web/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "emitDecoratorMetadata": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
Interesting, just read about it quickly. I guess it gives better bundling performance?

Reply:
I guess so. I primarily changed it to make sure packages' exports are supported and next requires this value